public/script.js

```diff
@@ -136,6 +147,20 @@ function nextDate() {
   if (STATE.current.isBefore(STATE.start)) {
     STATE.current= moment(STATE.start);
   }
+  let indexVal = 0;
+  if (STATE.extCon == 'concentration') {
+    indexVal += 4;
+  }
+  if (STATE.dateLoopeStyle == 'monthly') {
+    indexVal += 2;
+  }
+  if (STATE.norSouth == 's') {
+    indexVal +=1;
+  }
+  if (!(validDates[indexVal].includes(STATE.current.utc().startOf('day').toISOString()))) {
+    nextDate();
```

Comment:
Instead of skipping the missing data, just display "no data" for the missing data.